hamonize-center/sql/hamonize_center.sql

```diff
@@ -1340,7 +1340,7 @@ INSERT INTO public.tbl_svrlst (svr_nm,svr_domain,svr_ip,svr_dc,insert_dt,svr_por
 	 ('CENTERURL',NULL,'localhost',NULL,NULL,'8080',0,NULL),
 	 ('COLLECTDIP',NULL,'localhost',NULL,NULL,NULL,0,NULL),
 	 ('APTURL',NULL,'106.254.251.74',NULL,NULL,'28081',0,NULL),
-	 ('GRAFANA_URL',NULL,'localhost',NULL,NULL,NULL,0,NULL);
+	 ('GRAFANA_URL',NULL,'localhost',NULL,NULL,'3000',0,NULL);
```

Comment:
This chang is reasonable to me at least because a Grafanna's default port is 3000.

Acked-by: Geunsik Lim <leemgs@gmail.com>